### PR TITLE
Issue#545 Extend generated java doco to include private methods

### PR DESCRIPTION
**Pull request by abekornelis**

### CODE REVIEW

File: bat/BLDJVDOC.BAT [hunk 1]
```diff
@@ -15,7 +15,7 @@ pushd %~dps0..
 rem generate javadoc
 if not exist ".\docenv\javadoc\" mkdir docenv\javadoc
 cd docenv\javadoc
-call javadoc.exe ..\..\src\*.java || goto error
+call javadoc.exe ..\..\src\*.java -private || goto error
```
Comment (jyganci) [inline bat/BLDJVDOC.BAT:18]: Add similar change to bash/bldjavadoc. Line 20 there. New line 20 could be
```
javadoc -quiet -Xdoclint:none -d ${DEST} -private src/*.java
```

File: src/ez390.java [hunk 2]
```diff
@@ -294,7 +295,7 @@ public void process_ez390(String[] args,JTextArea log_text,JTextField command_te
     /**
      * execute IPL pgm and/or application pgm
      *
-     * @param zcvt_pgm_addr
+     * @param zcvt_pgm_addr - ??
```
Comment (jyganci) [inline src/ez390.java]: Why did you add " - ??"?

File: bat/BLDJVDOC.BAT (hunk 1 above)
Comment (abekornelis) [reply bat/BLDJVDOC.BAT:18]: Changed as per your suggestion. Thanks.

File: src/ez390.java (hunk 2 above)
Comment (abekornelis) [reply src/ez390.java]: The javadoc utility complained about missing description. I had no clue. So I put in question marks.
Admittedly a tad cryptic. I'll add proper text instead.

File: src/ez390.java [hunk 3]
```diff
@@ -294,7 +295,7 @@ public void process_ez390(String[] args,JTextArea log_text,JTextField command_te
     /**
      * execute IPL pgm and/or application pgm
      *
-     * @param zcvt_pgm_addr
+     * @param zcvt_pgm_addr - function of this parameter is currently unclear
```
Comment (jyganci) [inline src/ez390.java]: After looking in ez390 to see where run_pgm is invoked, ez390 line 297 should be something like
```
     * @param zcvt_pgm_addr  address of 8 byte field containing name of program to run
 ```

**Review by jyganci — APPROVED** [review]
